react/src/lib/components/DeckGLMap/layers/map/mapLayer.ts

```diff
@@ -303,47 +324,108 @@ async function load_mesh_and_texture(
             },
             indices: { value: new Uint32Array([0, 1, 3, 1, 3, 2]), size: 1 },
         };
-    }
 
-    const image_name = isTexture ? texture_name : mesh_name;
-    const texture = await load_texture(image_name); // load texture as ImageData structure.
+        const [w, h] = [2, 2];
+        meshData = new Float32Array([0, 0, 0, 0]);
+        meshData = makeFixedSizeCopy(meshData, w, h);
+    }
 
-    return Promise.all([mesh, texture]);
+    //-- PROPERTY TEXTURE. --
+    const response = await fetch(propertiesUrl);
+    if (!response.ok) {
+        console.error("Could not load ", propertiesUrl);
+    }
+    const buffer = await response.arrayBuffer();
+    const data = new Float32Array(buffer);
+
+    // create float texture.
+    const propertyValueRange = getMinMax(data);
+    const format = GL.R32F;
+    const type = GL.FLOAT;
+    const texture = new Texture2D(gl, {
+        width: w,
+        height: h,
+        format,
+        type,
+        data,
+        mipmaps: false,
+        parameters: DEFAULT_TEXTURE_PARAMETERS,
+    });
+
+    const propertyData = makeFixedSizeCopy(data, w, h);
+
+    return Promise.all([
+        mesh,
+        propertyValueRange,
+        meshData,
+        propertyData,
+        texture,
+    ]);
 }
 
 export interface MapLayerProps<D> extends ExtendedLayerProps<D> {
-    // Url to png image representing the height mesh.
-    mesh: string;
+    // Url to the height (z values) mesh.
+    meshUrl: string;
 
-    // Bounding box of the terrain mesh, [minX, minY, maxX, maxY] in world coordinates
-    bounds: [number, number, number, number];
+    // Bounding box of the terrain mesh [[origoX, origoY], [nx, ny], [dx, dy]]] in world coordinates.
+    // nx, y: number of cells in each direction. dx, dy: cell width in each direction.
+    bounds: [[number, number], [number, number], [number, number]];
```

Comment:
I think we should use the same bounds type and prop name as we decide on in https://github.com/equinor/webviz-subsurface-components/pull/1095